api/src/main/java/org/openmrs/validator/PersonValidator.java

```diff
@@ -128,11 +128,15 @@ private void validateBirthDate(Errors errors, Date birthDate) {
 	 * @param errors Stores information about errors encountered during validation
 	 * @param deathDate the deathdate to validate
 	 */
-	private void validateDeathDate(Errors errors, Date deathDate) {
+	private void validateDeathDate(Errors errors, Date deathDate, Date birthDate) {
 		if (deathDate == null) {
 			return;
 		}
 		rejectIfFutureDate(errors, deathDate, "deathDate");
+		if (birthDate == null) {
```

Comment:
@dkayiwa I have changed the error shown and also squashed the new commits into one :-)